bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/discovery/SomfyTahomaItemDiscoveryService.java

```diff
@@ -300,6 +300,9 @@ private void discoverDevice(SomfyTahomaDevice device, SomfyTahomaSetup setup) {
                     logUnsupportedDevice(device);
                 }
                 break;
+            case CLASS_WATER_HEATING_SYSTEM:
+                deviceDiscovered(device, THING_TYPE_WATERHEATINGSYSTEM, place);
+                break;
```

Comment:
Please use deeper detection using getWidget(), I guess there might be more water heating systems and this makes the autodetection more accurate. See the CLASS_CAMERA detection as an example...